pkg/notifications/notifications_test.go

```diff
@@ -27,3 +29,14 @@ func TestNotificationsSuite(t *testing.T) {
 
 	suite.Run(t, rs)
 }
+
+func (suite *NotificationsSuite) deserializeActivityMetadata(testActivity *models.Activity) models.ActivityMetaData {
```

Comment:
Optional: Could you add a comment to this test function? That would be helpful if the hope is to use this in other functions where the same logic is being tested.